django/thunderstore/api/cyberstorm/views/team.py

```diff
@@ -52,6 +62,58 @@ def get_queryset(self) -> QuerySet[TeamMember]:
         )
 
 
+class CyberstormTeamAddMemberRequestSerialiazer(serializers.Serializer):
+    user = serializers.CharField(max_length=User._meta.get_field("username").max_length)
+    role = serializers.ChoiceField(choices=TeamMember._meta.get_field("role").choices)
```

Comment:
The more appropriate field to bind this to would be the form field's choices, as the form is what the API view here is calling rather than the DB model